cpp/non_iid_main.cpp

```diff
@@ -377,27 +364,22 @@ int main(int argc, char* argv[]) {
         if (ret_min_entropy >= 0) {
             if (verbose == 1) printf("\tMulti Most Common in Window (MultiMCW) Prediction Test Estimate = %f / %d bit(s)\n", ret_min_entropy, data.word_size);
             H_original = min(ret_min_entropy, H_original);
+            tc637.ret_min_entropy = ret_min_entropy;
```

Comment:
Similarly, if both literal and bitstring assessments are conducted, only the literal assessment is saved.